sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/GcpApiSurfaceTest.java

```diff
@@ -49,6 +49,10 @@ public void testGcpApiSurface() throws Exception {
             .pruningPattern("org[.]apache[.]beam[.].*Test.*")
             .pruningPattern("org[.]apache[.]beam[.].*IT")
             .pruningPattern("org[.]checkerframework[.].*[.]qual[.].*")
+            // Quick check packages, exposed only in testing
```

Comment:
We added a generator `TimestampGenerator` which is used in testing only. The generator needs to be public, otherwise we get a `IllegalAccessException` from quickcheck.

Since these classes are used in test only, we thought is was ok to add them in exceptions here (like the `checkerframework` or other tests above)